apps/remix-ide/src/lib/helper.js

```diff
@@ -52,6 +50,27 @@ module.exports = {
   createNonClashingName (name, fileProvider, cb) {
     this.createNonClashingNameWithPrefix(name, fileProvider, '', cb)
   },
+  async checkNonClashingNameAsync (name, fileManager, prefix = '') {
```

Comment:
should it be `createNonClashingNameAsync` ?

Reply:
I can rename it to `createNonClashingNameAsync`. It is the same implementation but async without callbacks.